src/Itinero.IO.Osm/Streams/RouterDbStreamTarget.cs

```diff
@@ -299,7 +299,7 @@ public virtual void RegisterSource(OsmStreamSource source, bool normalize)
                         }
                         foreach (var tag in tags)
                         {
-                            if (_vehicleCache.Vehicles.IsOnMetaWhiteList(tag.Key))
+                            if (_vehicleCache.Vehicles.IsOnMetaWhiteList(tag.Key)&&!osmGeo.Tags.ContainsKey(tag.Key))
```

Comment:
Added checkpoint to avoid inserting duplicate tag.

Reply:
Perhaps `normalizedTags.Concat(tags).Distinct()`

Reply:
Perhaps also `ToHashSet` could be used instead... whatever makes more sense